### DISCUSSION TITLE
Need away to disable faker

### DISCUSSION THREAD

**Discussion by samuraitruong** [mikro-orm/mikro-orm#3089] [Q&A]
**Is your feature request related to a problem? Please describe.**
 - We develop application in docker on local host and application restart overtime code change.  When app restarted it will trigger full reload all module incude @mikro-orm/seeder. But seeder took too much time for load faker lib. In my app, we dont need to use any faker data so we want the feature that can disable faker lib loading

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context about the feature request here.

**Comment by B4nan** [comment]
I have no idea what you mean by "disable faker lib loading". There is no initialization, so you are telling me that just the import of faker itself is problematic? Because you are the first one saying this, in ORM tests there is no perf penalty for that.

How did you even measure this, why do you think this is the culprit? Are you using latest versions?

**Reply by samuraitruong** [reply]
Hi @B4nan,
f you run this inside docker container and just put console log before and after. 'import @mikro-orm/seeder` it will take 40 sec just for load the library. I accepted the fact that volume mount make docker become very slow but most of other library only took <  1 sec include @mikro-orm/core. The faker is dependencies and always loaded with seeder. I don't need to user any faker so I wish to not include this library. I can fork the library and remove it to use in our project however, it will be great if that become the function support by default.

Yes, I am on latest version


Note: If you run the code on host machine it will very fast and you may not aware of it . (development in docker with volume mount make it slower 60x)

**Reply by alexojegu** [reply]
From the `fakerjs` guide about [localization](https://fakerjs.dev/guide/localization.html#individual-localized-packages):
> By default, Faker will include all locale data.
> This might result in loading around 5 MB of data into memory and slow down startup times.
> 
> But we got your back!
> When encountering such a problem in a test or production environment, you can use the individual localized packages.

Could be possible a mikro-orm config option to import only the desired locale? for example something as:
```
export default {
  type: 'postgresql',
  seeder: {
    pathTs: 'seeders',
    locale: 'es',
  },
};
```

**Reply by B4nan** [reply]
That's huge. I guess we can just drop the faker dependency in v6 and let people install it explicitly where needed, we don't "use" it anyhow, it's basically just a re-export that is also passed in the factory parameters.

**Reply by B4nan** [reply]
Removed from v6 here:

https://github.com/mikro-orm/mikro-orm/commit/1a795e8889f7d93737a6e433cd75157ee12c979d
